rust/xaynet-analytics/src/database/analytics_event/data_model.rs

```diff
@@ -58,4 +58,11 @@ impl<'a> IsarAdapter for AnalyticsEvent<'a> {
         object_builder.write_string(self.screen_route.map(|screen| screen.name.as_ref()));
         object_builder.write_string(Some(&self.timestamp.to_rfc3339()));
     }
+
+    fn read(_bytes: &[u8]) -> AnalyticsEvent<'screen> {
+        // TODO: implement when Isar will support it: https://xainag.atlassian.net/browse/XN-1604
+        unimplemented!()
+    }
 }
+
+impl<'screen> SchemaGenerator<AnalyticsEvent<'screen>> for AnalyticsEvent<'screen> {}
```

Comment:
is this ok to just use the default implementation? or are there better ways in Rust?

Reply:
if the struct is supposed to use the default impl of the trait, then this is exactly the way to go (even though it might look a little weird to have an "empty" impl block).

Reply:
ok thanks, sounds good